scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -17,6 +17,13 @@ workflow GvsExtractCallset {
         String? fq_filter_set_site_table
         String? fq_filter_set_tranches_table
         String? filter_set_name
+
```

Comment:
Can we make the corresponding changes above to default these table names given a "default_dataset" etc?  We can also make them non-optional, though we would need to check that works with ExtractCohort (ie if I specify those table names, but no filter set... I mean "don't filter"... we might explode right now in that scenario)

Reply:
yeah, I'll give it a shot, we should think about what the decision-making input will be that decides whether to filter or not. right now it's whether `--filter-set-info-table` is input to the tool - maybe we should change that to be whether a filter set name is input? then we could have defaults for the table names like you suggest

Reply:
@kcibul how does this look to you now? i tested that it works with and without filtering in Terra (i.e. providing filter-table-name, and setting do_not_filter_override to true, respectively)